azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/AppServiceUtils.java

```diff
@@ -109,4 +110,20 @@ public static AppServicePlan updateAppServicePlan(final AppServicePlan appServic
     public static boolean isEqualAppServicePlan(AppServicePlan first, AppServicePlan second) {
         return first == null ? second == null : second != null && StringUtils.equals(first.id(), second.id());
     }
+
+    public static DockerImageType getDockerImageType(final String imageName, final String serverId,
+                                                     final String registryUrl) {
+        if (org.apache.commons.lang3.StringUtils.isEmpty(imageName)) {
```

Comment:
use only one of org.codehaus.plexus.util.StringUtils or org.apache.commons.lang3.StringUtils in this class

Reply:
Good catch, will fix it in later pr.